pkg-manager/plugin-commands-installation/src/dedupe.ts

```diff
@@ -57,7 +60,6 @@ export async function handler (opts: DedupeCommandOptions) {
   return installDeps({
     ...opts,
     dedupe: true,
-    ignoreScripts: opts.ignoreScripts ?? false,
```

Comment:
This can be removed since the `...opts` object above includes `ignoreScripts` now. There's a test for this that was previously added by another helpful contributor.